### ISSUE TITLE
Put your ideas on what more can be added to this !!

### ISSUE THREAD

**Issue by Harshita248** [Harshita248/WOOGLE#2]
Any new feature or enhancement for this !!

**Comment by aniketsharma00411** [comment]
Can you first explain the project?

**Comment by Harshita248** [comment]
Wanted to make a search box which have all types of filter and also improve the UI of the website overall ...and also it can be linked with google so that search appears @aniketsharma00411

**Comment by Himanshu-Modi** [comment]
There are several enhancement possible-
1.Currently It is showing only video searches . So  Tag line could be- Woogle- Video Searching Platform.
2. Size of search button could be enhanced.
3.If the result cold be wrapped inside a card and click to watch button could be added.
Can i work on above suggested points.

**Comment by Harshita248** [comment]
Yes sure please work on the ideas specified by you @Himanshu-Modi

**Comment by dev-abir** [comment]
we can use google dorks for filter, not sure if it's legal

**Comment by Harshita248** [comment]
Yes and if you want to add that please go ahead @dev-abir . You can also add more features to it just look for the same add and list in the commits with screenshots

**Comment by dev-abir** [comment]
are u sure, it's legal?

**Comment by Harshita248** [comment]
no

**Comment by dev-abir** [comment]
well, so how should we add filters?
